src/assets/css/style.css

```diff
@@ -1,9 +1,149 @@
 body {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
     Arial, sans-serif;
   margin: auto;
   max-width: 38rem;
   padding: 2rem;
-  background: linear-gradient(to right, yellow, red)
+  background: linear-gradient(to right, yellow, red);
 }
 
+
+/*-------------CALL INTERFACE BUTTON---------- */
+:root{
```

Comment:
This is suppose to be at the beginning of the css file most times.
You should pull before you push